Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -11,45 +10,54 @@ Firstly you will need to select the parent page for your new page. This could be
 
 1. Hover over the name of the parent page and then click on the options icon (•••) to reveal the types of pages you can create.
 2. Click on the type of page you wish to create which will then load the new page into the content area on the right-hand side.
-3. Enter a name for the page in the text field at the top of the content area. See sections 2.3.2 and 2.5 for more information on saving and editing the new page.
+3. Enter a name for the page in the text field at the top of the content area. 
 
-![pageCreationSlidout.jpg](images/pageCreationSlidout.jpg)
+![This shows an arrow pointing at the home node in the section tree.](images/Step-one-parent-page.png)
 
-![blankPage.jpg](images/blankPage.jpg)
+![This shows an arrow pointing at Enter a name...](images/Enter-the-page-name-here.png)
 
 ## Various Saving and Publishing Options
 
 There are four different options you have for saving and publishing pages. Each have their own benefits for different occasions depending on whether you’re still in the process of editing the page or have completed and wish to publish your changes.
 
 ### Save
 
-The ***Save*** option (found by clicking the arrow on the ***Save and publish*** button) is used for saving the page without publishing the changes to the live site. This is especially useful if you are working on changes over a period of time as you can save frequently to prevent losing any data or if you wish to save the current state and return to edit the page at a later date. Using the ***Preview*** button will show you how the page will look once published.
+The ***Save*** option (Located next to the ***Save and publish*** button) is used for saving the page without publishing the changes to the live site. This is especially useful if you are working on changes over a period of time as you can save frequently to prevent losing any data or if you wish to save the current state and return to edit the page at a later date. Using the ***Preview*** button will show you how the page will look once published.
```

Comment:
This sentence was a bit too long - I made some suggestions. What do you think?

```suggestion
The ***Save*** option (Located next to the ***Save and publish*** button) is used for saving the page without publishing the changes to the live site. This is especially useful if you are working on changes over a period of time as you can save frequently to prevent losing any data. Saving the content page will also enable you to use the ***Preview*** feature, which will show you how the page will look once published. The Preview feature is accessed next to the Save button.
```